File: plugins/OrchidJavadoc/src/main/java/com/eden/orchid/javadoc/JavadocResourceSource.java
package com.eden.orchid.javadoc;

import com.eden.orchid.api.OrchidContext;
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource;

import javax.inject.Inject;

public class JavadocResourceSource extends PluginResourceSource {

    @Inject
    public JavadocResourceSource(OrchidContext context) {
        super(context, 10);
    }
}

File: plugins/OrchidJavadoc/src/main/kotlin/com/eden/orchid/javadoc/JavadocResourceSource.kt
package com.eden.orchid.javadoc

import com.eden.orchid.api.OrchidContext
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource
import javax.inject.Inject

class JavadocResourceSource @Inject
constructor(context: OrchidContext) : PluginResourceSource(context, 10)
